deployments/infrastructure-target-aws.nix

```diff
@@ -17,7 +17,9 @@ in rec {
       associatePublicIpAddress = true;
       securityGroups = [
         resources.ec2SecurityGroups."allow-deployer-ssh-${region}-${org}"
-        resources.ec2SecurityGroups."allow-public-www-${region}-${org}"
+        resources.ec2SecurityGroups."allow-hydra-ssh-${region}-${org}"
+        resources.ec2SecurityGroups."allow-public-www-https-${region}-${org}"
+        resources.ec2SecurityGroups."allow-public-www-http-${region}-${org}"
```

Comment:
this fixed being unable to open http://hydra.iohk.io to get the redirect
and also hydra being unable to ssh its own slaves